confidential/secrets_manager.py

```diff
@@ -12,8 +12,8 @@
 
 
 class SecretsManager:
-    def __init__(self, secrets_file=None, secrets_file_default=None, region_name=None):
-        session = boto3.session.Session()
```

Comment:
Aside, but I had thought of moving the region parameter to the [session instantiation](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/session.html#boto3.session.Session) but didn't want to change too much.